src/frontend/ts/types/urbitChess.ts

```diff
@@ -81,36 +81,39 @@ export type SAN = string
 
 export type FENPosition = string
 
-export type Move = {
+export interface Move {
   from: CgKey
   to: CgKey
   san: SAN
   fen: FENPosition
 }
 
-export type GameInfo = {
+export interface GameInfo {
   gameID: GameID,
   event: string,
   site: string,
   round: string,
   white: Ship,
   black: Ship,
   result: Result,
```

Comment:
`result` should be removed from here and moved into `ArchivedGameInfo`

Reply:
that makes so much sense!